openedx/features/job_board/views.py

```diff
@@ -18,3 +20,12 @@ class JobDetailView(DetailView):
     model = Job
     template_engine = 'mako'
     template_name = 'features/job_board/job_detail.html'
+
+
+class JobCreateView(CreateView):
+    model = Job
+    fields = '__all__'
+    template_name = 'features/job_board/create_job_form.html'
+
+    def get_success_url(self):
+        return reverse('job_list')
```

Comment:
Wouldn't this work here: `success_url = reverse('job_list')`?

Reply:
This is not working here, I searched about it but it's attribute of FormView.